CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Next Release
 -------------
+1.8.2
+------
+* Add generic options object to App init. #46
```

Comment:
this would be a 1.9 feature release. Also, we need README documentation on how to use this new feaature.

Reply:
will update it.